app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<pages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/PageObject.xsd">
+    <page name="AdminNewsletterConfigPage" url="admin/system_config/edit/section/newsletter/" area="admin" module="Magento_Newsletter">
```

Comment:
You don't need to provide `admin/` at the beginning.

Reply:
Hi @lbajsarowicz. 
After changes  `url="admin/system_config/edit/section/newsletter/"` to `url="system_config/edit/section/newsletter/"` 

I am getting such issue. I can't visit config page correctly.
<img width="1273" alt="Screenshot 2020-01-27 at 13 38 03" src="https://user-images.githubusercontent.com/13517181/73172516-de679f00-410b-11ea-91cb-38e0e1941508.png">

Also I couldn't find examples in core without `admin/`
Could You Please advise me something about it. If you can.

Thanks.

Reply:
OK. I'm sorry, revert `admin/` then. I'll take a look into it later :-)